table/tables/cache.go

```diff
@@ -203,3 +230,34 @@ func (c *cachedTable) RemoveRecord(ctx sessionctx.Context, h kv.Handle, r []type
 	}
 	return c.TableCommon.RemoveRecord(ctx, h, r)
 }
+
+func (c *cachedTable) RenewLease() {
```

Comment:
It's not used outside of this package, is it necessary to expose this function?

Reply:
yes